.github/workflows/container.yml

```diff
@@ -0,0 +1,64 @@
+name: Container Image Builds
+
+on:
+  push:
+    branches: [ main ]
+    tags: ["v*"]
+  workflow_dispatch:
+
+
+jobs:
+  images:
+    name: Build and upload container images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Find reference version
+        uses: greenbone/actions/reference-version@v1
+      - name: Login to Docker Registry
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and push Container image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          tags: ${{ github.repository }}:${{ env.VERSION }}
+          file: .docker/stable.Dockerfile
```

Comment:
There is no `stable.Dockerfile`. Either we use a common build file for all branches and or you have to rename the `main.Dockerfile`